### DISCUSSION TITLE
Modify max TPS of server

### DISCUSSION THREAD

**Discussion by onLaait** [Minestom/Minestom#1741] [Help] [answered]
Hi, how can I modify max TPS of server?

**Comment by ZakShearman** [comment, marked as answer]
You can set the property `minestom.tps` (see here https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/MinecraftServer.java#L59)

Note that there normally isn't much use to increasing TPS. There is also a technical limit of 1000 TPS as the TPS is simply converted to `TICK_MS` with the minimum therefore being 1ms.

**Reply by mworzala** [reply]
I will also add to this, there is no guarantees of things working or being fixed using this. It's not a well tested use case :)
